kernel/src/syscall/execve.rs

```diff
@@ -67,6 +60,13 @@ fn lookup_executable_file(
 
     let path = {
         let filename = filename.to_string_lossy();
+        if filename.is_empty() && !flags.contains(OpenFlags::AT_EMPTY_PATH) {
+            return_errno_with_message!(
+                Errno::ENOENT,
+                "filename is empty while AT_EMPTY_PATH is not specified"
+            );
+        }
```

Comment:
No need to do this now. `from_fd_and_path` will return errors if the file path is empty.